mscore/pianoroll/pianoroll.cmake

```diff
@@ -16,3 +16,7 @@ set(PIANOROLL_SRC
     ${CMAKE_CURRENT_LIST_DIR}/pianoview.cpp
     ${CMAKE_CURRENT_LIST_DIR}/pianoview.h
     )
+
+set(MSCORE_MIXER_UI
```

Comment:
No doubt you meant to write
```
set(PIANOROLL_UI
```
Otherwise, you will overwrite the MSCORE_MIXER_UI variable, and `ui_mixer.h` will not be generated. Then, in order to make sure that `ui_pianolevelschooser.h` is generated, you will need to make use of the PIANOROLL_UI variable in `mscore/CMakeLists.txt`. See https://github.com/mattmcclinch/MuseScore/commit/ff18b1e4a5c0ddb76029eb96c49270b6926038f1

Reply:
That worked.  Thanks.